include/assembler.cpp

```diff
@@ -53,10 +54,21 @@ uint16_t swap_bytes(uint16_t number) {
     return result;
 }
 
-void assembleLine(std::string line, int lineNum, std::string romName) {
+int assembleLine(const std::string& line, int lineNum, const std::string& romName) {
```

Comment:
I feel like `line` is a bit unclear. Maybe change to `codeLine` or `mnemonicLine` or similar.

Also as said via discord, passing through `romName` feels odd.